src/reducers/app.js

```diff
@@ -145,6 +145,7 @@ const panelLayoutGeneration: Reducer<number> = (state = 0, action) => {
     case 'POP_COMMITTED_RANGES':
     // Bottom box: (fallthrough)
     case 'OPEN_SOURCE_VIEW':
```

Comment:
nit: this `OPEN_SOURCE_VIEW` was forgotten

Reply:
Good catch!